CHANGELOG.md

```diff
@@ -13,24 +13,7 @@ and the [opentelemetry-collector-contrib v0.139.0](https://github.com/open-telem
 ### ❗ Known Issues ❗
 
 - (Splunk) `quay.io`: Windows Docker images aren't available at [quay.io](https://quay.io/repository/signalfx/splunk-otel-collector) yet. This is actively being investigated.
-
-## v0.139.1
-
-This Splunk OpenTelemetry Collector release includes changes from the [opentelemetry-collector v0.139.0](https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.139.0)
-and the [opentelemetry-collector-contrib v0.139.0](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.139.0) releases where appropriate.
-  
-### ❗ Known Issues ❗
-
-- (Splunk) `quay.io`: Windows Docker images aren't available at [quay.io](https://quay.io/repository/signalfx/splunk-otel-collector) yet. This is actively being investigated.
-
-## v0.139.0
-
-This Splunk OpenTelemetry Collector release includes changes from the [opentelemetry-collector v0.139.0](https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.139.0)
-and the [opentelemetry-collector-contrib v0.139.0](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.139.0) releases where appropriate.
-
-### ❗ Known Issues ❗
-
-- (Splunk) `quay.io`: Windows Docker images aren't available at [quay.io](quay.io/repository/signalfx/splunk-otel-collector).
+- (Splunk) `quay.io`: the tags `0.139.0*` and `0.139.1*` published to [quay.io](https://quay.io/repository/signalfx/splunk-otel-collector) for non-Windows platforms were built with the same code as `v0.139.2` and share the same changelog.
```

Comment:
Did you want to make *`0.139.0`* and *`0.139.1`*? Otherwise * can be removed I guess